src/js/view/controls/controls.js

```diff
@@ -110,7 +110,11 @@ export default class Controls extends Events {
                 this.trigger(DISPLAY_CLICK);
                 this.userActive(1000);
                 api.playToggle(reasonInteraction());
-                model.get('isFloating') ? this.wrapperElement.focus() : this.playerContainer.focus();
+                if (model.get('isFloating')) {
+                    this.wrapperElement.focus();
+                } else {
+                    this.playerContainer.focus();
+                }
```

Comment:
We're doing the same thing on line 214 with:
```
const focusElement = model.get('isFloating') ? this.wrapperElement : this.playerContainer;
focusElement.focus();
```

Reply:
good call! I'll keep a look out for things like this.